packages/html/src/lib.rs

```diff
@@ -35,6 +36,7 @@ pub use transit::*;
 pub use elements::*;
 pub use events::*;
 pub use global_attributes::*;
+pub use point_interaction::PointInteraction;
```

Comment:
I agree this would be nice in the prelude. We could add this to the html prelude and it will be re-exported in the dioxus prelude automatically

Reply:
added to prelude